pkg/proc/native/proc.go

```diff
@@ -372,7 +372,7 @@ func (dbp *nativeProcess) initialize(path string, debugInfoDirs []string) (*proc
 		//	  with gdb once AsyncPreempt was enabled. While implementing the port,
 		//	  few tests failed while it was enabled, but cannot be warrantied that
 		//	  disabling it fixed the issues.
-		DisableAsyncPreempt: runtime.GOOS == "windows" || (runtime.GOOS == "linux" && runtime.GOARCH == "arm64") || (runtime.GOOS == "linux" && runtime.GOARCH == "ppc64le"),
+		DisableAsyncPreempt: runtime.GOOS == "windows" || (runtime.GOOS == "linux" && runtime.GOARCH == "arm64") || (runtime.GOOS == "linux" && runtime.GOARCH == "ppc64le") || (runtime.GOOS == "linux" && runtime.GOARCH == "riscv64"),
```

Comment:
There is a comment above explaining why we disable async preempt, if it really needs to be disable on riscv64 it should be updated to describe why.

Reply:
Some tests failed like the TestTraceDirRecursion, the symptom which seems to cause by async preempt but disable it doesn't work just like I said, so I'll enable it, until we find the exact reason.

> Most of tests were passed, the few failed tests seems to be related with riscv64 go compiler itself, for example, the strange repetition in `TestTraceDirRecursion`, just like this:
> 
> ```
> > goroutine(1): main.A(5, 5)
> > goroutine(1): main.A(4, 4)
> > goroutine(1): main.A(4, 4)
> > goroutine(1): main.A(3, 3)
> > goroutine(1): main.A(2, 2)
> > goroutine(1): main.A(2, 2)
> > goroutine(1): main.A(1, 1)
> > goroutine(1): main.A(1, 1)
> >> goroutine(1): main.A => (1)
> >> goroutine(1): main.A => (2)
> >> goroutine(1): main.A => (6)
> >> goroutine(1): main.A => (24)
> >> goroutine(1): main.A => (120)
> 120
> ```
> 
> I tried to disable the asyncpreemt, but unfortunately, It doesn't work.